packages/core/src/components/FileUploader/File/File.styles.tsx

```diff
@@ -3,22 +3,12 @@ import { theme } from "@hitachivantara/uikit-styles";
 
 export const { staticClasses, useClasses } = createClasses("HvFile", {
   root: {},
-  progressbar: {
+  progressBar: {
```

Comment:
I would leave the class names as they were because changing them will lead to breaking changes if someone is using them.